orttraining/orttraining/core/session/training_session.cc

```diff
@@ -154,9 +154,7 @@ Status TrainingSession::ConfigureForTraining(
                                          config.distributed_config.horizontal_parallel_size,
                                          config.distributed_config.pipeline_parallel_size});
 
-  const int32_t pipeline_stage_id = config.pipeline_config.has_value() ?
-                              DistributedRunContext::RankInGroup(WorkerGroupType::ModelParallel) :
-                              -1;
+  const int32_t pipeline_stage_id = config.pipeline_config.has_value() ? DistributedRunContext::RankInGroup(WorkerGroupType::ModelParallel) : -1;
```

Comment:
nit: line lengths - i wonder if breaking the line before the `?` will be more acceptable to the formatter?

Reply:
I decide to revert it since it's a one line change.